### PR TITLE
add Player::getDeviceID,Player::getDeviceOS

### PR DESCRIPTION
**Pull request by Blackjack200**
add Player::getDeviceID,Player::getDeviceOS

### CODE REVIEW

File: src/pocketmine/Player.php [hunk 1]
```diff
@@ -385,6 +385,13 @@ public static function isValidUserName(?string $name) : bool{
 	/** @var Vector3|null */
 	protected $lastRightClickPos = null;
 
+	/** @var string */
+	protected $deviceId;
+	/** @var string */
+	private $deviceModel;
+	/** @var int */
+	private $deviceOS;
```
Comment (TwistedAsylumMC) [inline src/pocketmine/Player.php:393]: Why is deviceId protected and device model and deviceOS private?

File: src/pocketmine/Player.php [hunk 2]
```diff
@@ -4078,4 +4089,16 @@ public function getLoaderId() : int{
...
+		return $this->deviceOS;
+	}
+
+	public function getDeviceModel() : string{
+		return $this->deviceModel;
+	}
+
+	public function getDeviceId() : string{
+		return $this->deviceId;
+	}
```
Comment (TwistedAsylumMC) [inline src/pocketmine/Player.php:4103]: These are in reverse order of the properties

**Comment by TwistedAsylumMC** [comment]
As well as that, you also did not state you added the "getDeviceModel" method. This is also still targeted at the "stable" branch so the same response from your last PR still applies:
> In addition, this PR is targeted at the stable branch, which is not accepting changes apart from protocol changes and bugfixes.

**Comment by dktapps** [comment]
...

**Comment by mmm545** [comment]
dktapps: *sigh*

**Pull request closed without merging**